app/app.go

```diff
@@ -439,7 +439,11 @@ func (a *App) WaitForGoroutines() {
 }
 
 func (a *App) HTMLTemplates() *template.Template {
-	return a.htmlTemplateWatcher.Templates()
+	if a.htmlTemplateWatcher != nil {
```

Comment:
Strictly speaking, this shouldn't happen anymore given the `FindDir` changes, but it helps pinpoint future problems vs. segfaulting.